great_expectations/checkpoint/actions.py

```diff
@@ -297,10 +296,15 @@ def _run(  # type: ignore[override] # signature does not match parent  # noqa: P
                 "store_validation_result" in payload
                 and "validation_result_url" in payload["store_validation_result"]
             ):
-                validation_result_urls.append(
+                cloud_validation_url = (
                     payload["store_validation_result"]["validation_result_url"]
+                    + "?slack=true"
```

Comment:
What is this string? Could we add it as a constant or classvar and give it an appropriate name?

```python
MY_SLACK_QUERY_PARAM = "?slack=true"

...

                cloud_validation_url = (
                    payload["store_validation_result"]["validation_result_url"]
                    + cls.MY_SLACK_QUERY_PARAM
```